braindecode/augmentation/transforms/masking_along_axis.py

```diff
@@ -0,0 +1,138 @@
+import torch
+
+FFT_ARGS = {"n_fft": 512, "hop_length": 256,
+            "win_length": 512}
+DATA_SIZE = 3000
+
+
+def mask_along_axis(X, params):
+    r"""
+    Apply a mask along ``axis``. Mask will be applied from indices
+    ``[v_start, v_end)``, .
+    All examples will have the same mask interval.
+
+    Args:
+        X (Tensor): Real spectrogram (channel, freq, time)
+        mask_start (int): First column masked
+        mask_end (int): First column unmasked
+        mask_value (float): Value to assign to the masked columns
+        axis (int): Axis to apply masking on (1 -> frequency, 2 -> time)
+
+    Returns:
+        Tensor: Masked spectrogram of dimensions (channel, freq, time)
+    """
+
+    mask_start = params["mask_start"]
+    mask_end = params["mask_end"]
+
+    if params["axis"] == 1:
+        X[:, mask_start:mask_end, :, :] = params["mask_value"]
+    elif params["axis"] == 2:
+        X[:, :, mask_start:mask_end, :] = params["mask_value"]
+    else:
+        raise ValueError('Only Frequency and Time masking are supported')
+    X = X.reshape(X.shape[:-2] + X.shape[-2:])
+    return X
+
+
+def mask_along_axis_random(X, params):
+    """Given a magnitude and an axis, produces a masking interval
+    ``[v_start, v_end)`` where ``v_end - v_start`` is sampled from
+     ``uniform(0, magnitude * v_max)`` and rounded,
+    and ``v_start`` from ``uniform(0, v_max - v)`` and rounded.
+
+    Args:
+        X (Tensor): data
+        params (dict): a dict containing the necessary parameters for applying
+            the transform : the magnitude, the axis and the masking value.
+
+    Returns:
+        Tensor: Masked data
```

Comment:
your docstring formats do not follow the numpydoc standard.

see an example:

https://github.com/braindecode/braindecode/blob/master/braindecode/classifier.py#L190

with Parameters and Returns section.

you could run pydocstyle of your doc to check these things. https://pypi.org/project/pydocstyle/